controlplane/src/core/bufservices/organization/leaveOrganization.ts

```diff
@@ -36,7 +36,7 @@ export function leaveOrganization(
 
     const orgMember = await orgRepo.getOrganizationMember({
       organizationID: authContext.organizationId,
-      userID: authContext.userId || req.userID,
+      userID: req.userID || authContext.userId,
```

Comment:
I have inverted this as the `authContext.userId` is always present so, if the `userId` is provided on the request, it was never being looked at